src/pubsub.ts

```diff
@@ -36,13 +21,15 @@ export class PubSub implements PubSubEngine {
     }
 
     public publish(triggerName: string, payload: any): boolean {
-        this.ee.emit(triggerName, payload);
+        process.nextTick(() => this.ee.emit(triggerName, payload));
```

Comment:
I dont remember the exact issue, but it is related to how the tests assume data is handled.. i can dig in again